### ISSUE TITLE
Voiceover: Map interactivity doesn't work when voiceover is enabled

### ISSUE THREAD

**Issue by nickom** [sunlightlabs/congress-ios#235]
Tapping to drop a pin, pinching to zoom and dragging to move the map doesn't seem to work when voiceover is on. It highlights the map box and reads the information but you can't actually utilize the functions mentioned. 

Note: it is likely I'm just not familiar enough with voiceover gestures so someone else should confirm this issue.
